Firestore/Example/Tests/Local/FSTLevelDBMigrationsTests.mm

```diff
@@ -18,17 +18,18 @@
 #include <leveldb/db.h>
 
 #import "Firestore/Protos/objc/firestore/local/Target.pbobjc.h"
+#import "Firestore/Source/Local/FSTLevelDB.h"
 #import "Firestore/Source/Local/FSTLevelDBKey.h"
 #import "Firestore/Source/Local/FSTLevelDBMigrations.h"
 #import "Firestore/Source/Local/FSTLevelDBQueryCache.h"
-#import "Firestore/Source/Local/FSTWriteGroup.h"
 
 #include "Firestore/core/src/firebase/firestore/util/ordered_code.h"
 
 #import "Firestore/Example/Tests/Local/FSTPersistenceTestHelpers.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
+using firebase::firestore::local::LevelDBTransaction;
```

Comment:
This must now be broken: `s/LevelDBTransaction/LevelDbTransaction/`.

Reply:
Fixed.